docs/user-guide/sync_retry_options.md

```diff
@@ -0,0 +1,248 @@
+# Sync Retry Options
+
+Argo CD provides configurable retry mechanisms to handle sync failures gracefully. When enabled, retries can help resolve transient issues automatically, reducing manual intervention and improving application reliability. This document explains how retry options work, their configuration, and behavior in different scenarios.
+
+## Overview
+
+Sync retry options in Argo CD control whether failed sync operations are automatically retried. This feature is particularly useful in production environments where transient failures (such as network issues, temporary resource constraints, or cluster unavailability) can cause sync operations to fail.
+
+By configuring retry options appropriately, you can:
+- Automatically recover from transient failures
+- Reduce manual intervention during sync operations
+- Improve application deployment reliability
+- Handle cluster resource constraints gracefully
+
+## Retry Option States
+
+### Retry OFF
+- **Behavior**: No automatic retries on sync failures
+- **Use Case**: When you want full control over sync operations
+- **Timing**: Failed syncs wait until the next scheduled sync interval or manual sync
+- **Best For**: Debugging sync issues, development environments, or when you need complete oversight
+
+### Retry ON
+- **Behavior**: Automatically retry failed sync operations
+- **Use Case**: When you want to handle transient failures automatically
+- **Timing**: Retries occur based on configured backoff settings until success or limit reached
+- **Best For**: Production environments, high-availability requirements, or when minimizing manual intervention
+
+## Sync Intervals and Retries
+
+### Automated Sync
+- **Default Interval**: 3 minutes
+- **Behavior**: Argo CD checks Git repository for changes at regular intervals
+- **With Retry OFF**: Failed syncs wait for next interval
+- **With Retry ON**: Failed syncs retry automatically based on retry configuration
+
+### Manual Sync
+- **Trigger**: User-initiated sync operations
+- **Behavior**: Immediate sync attempt regardless of retry settings
+- **Use Case**: When you need immediate sync without waiting for intervals
+
+## Configuration
+
+### Basic Retry Configuration
+
+Retry options are configured in the Application resource under `spec.syncPolicy`:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: my-application
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/my-repo.git
+    path: manifests
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: default
+  syncPolicy:
+    automated:
+      prune: true
+      selfHeal: true
+    retry:
+      limit: 5
+      backoff:
```

Comment:
Didn't you want to use a different example here? Exponential backoff is explained next section.